src/pages/RealmExplorer.js

```diff
@@ -73,8 +74,8 @@ const getRealmObjectsFields = ({ schema: objectSchemas }) =>
     })
     .reduce((acc, object) => ({ ...acc, ...object }));
 
-const REALM_OBJECTS = getRealmObjects(schema);
 const REALM_OBJECTS_FIELDS = getRealmObjectsFields(schema);
+const REALM_OBJECTS = Object.keys(REALM_OBJECTS_FIELDS);
```

Comment:
The `REALM_OBJECTS_FIELDS` has correct key names, which relate to table names.
The previous `REALM_OBJECTS` is giving minified versions of the properties, which is causing havoc.